HHS/Head-Start-TTADP
# [TTAHUB-870] Script to index existing AR's (CircleCI Opensearch Image)

## Description of change

This script (job) deletes the current index for activity reports (if it exists). Then it recreates the activity report index with documents for: 

- AR Context
- Goal Name
- Objective Title and TTA
- Specialist Next Steps
- Recipient Next Steps

**Note:** This script was written with raw SQL as Sequelize reached a memory limit.

## How to test

- Review all code
- Make sure the unit tests passes.
- As a final test you can run the job against a clean DB to make sure it completes without (we should be able to run the job multiple times without error).

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-870


## Checklists

### Every PR



### Production Deploy


### After merge/deploy
